cvat/apps/engine/media_extractors.py

```diff
@@ -596,7 +596,7 @@ def _compress_image(source_image: av.VideoFrame | io.IOBase | Image.Image, quali
             with Image.open(source_image) as _img:
                 image = ImageOps.exif_transpose(_img)
         elif isinstance(source_image, Image.Image):
-            image = source_image
+            image = ImageOps.exif_transpose(source_image)
```

Comment:
Probably, we also need to handle EXIF in https://github.com/opencv/cvat/pull/7162/files#diff-d880e479fb2f54d4dd7603b1383ac44756996e153d86ecb6e4f3f7490b5a8505R706-R709 for correct `w` and `h` values.

Reply:
Discussed this in a call